pyverbs/cmid.pyx

```diff
@@ -65,6 +65,10 @@ cdef class ConnParam(PyverbsObject):
     def private_data(self):
         return <object>self.conn_param.private_data
 
+    def set_private_data(self, data, data_len):
```

Comment:
I suggest that you add the option to set the private data in the __init__

Reply:
@EdwardSro Add parameter data_len in the **init**